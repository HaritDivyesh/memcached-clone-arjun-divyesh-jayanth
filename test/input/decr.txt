1
flush_all
2
set mykey 1 0 2
hi
1
decr mykey 3
2
set key 1 0 2
50
1
decr key 5
2
set key 1 0 2
99
1
decr key -5
1
quit
