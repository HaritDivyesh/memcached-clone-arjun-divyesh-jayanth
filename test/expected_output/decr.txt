flush_all
OK
set mykey 1 0 2
hi
STORED
decr mykey 3
CLIENT_ERROR cannot increment or decrement non-numeric value
set key 1 0 2
50
STORED
decr key 5
45
set key 1 0 2
99
STORED
decr key -5
CLIENT_ERROR invalid numeric delta argument
quit
